### PR TITLE
update on ig form

### PR DESCRIPTION
**Pull request by senseibara**

### CODE REVIEW

**Comment by senseibara** [comment]
Could you please check the tests @kennsippell ? Weird things are happening. 

In the investigation form the once field in 133. is relevant only `if${validation_bis} = 'valid'` I added this because in the UI the true value was already assigned before we get to the details group ( i added a validation group just before ). But it makes a bunch of tests failing. Deleting the relevant condition make the tests pass but the informations data are not copied in the investigation form when i test manually ( which is weird because the automated test should fail then too ). 

Is it a limitation with the test-harness ? How can we work around it ?

**Comment by kennsippell** [comment]
https://github.com/medic/medic-conf-test-harness/issues/61

File: niger/app_settings.json [hunk 1]
```diff
@@ -308,8 +308,8 @@
     "death_reporting": false
   },
   "purge": {
-    "run_every_days": 7,
-    "cron": "20 14 * * mon",
+    "run_every_days": 14,
+    "cron": "0 20 * * sat",
```
Comment (kennsippell) [inline niger/app_settings.json:312]: intentional?

File: niger/test/tasks/investigation.spec.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ describe('Investigation task', () => {
     for(let i=0;i<2;i++) {
 
       await harness.loadAction(investigationTasks[1-i].actions[0]);
-      const investigationResult = await harness.fillForm(...investigation.unmodifiedInvalid);
+      const investigationResult = await harness.fillForm(...investigation.sampleNotCollected);
```
Comment (kennsippell) [inline niger/test/tasks/investigation.spec.js:66]: The goal of this test is to validate that the patient's defaults captured on the rgRiposteFollowup form are captured and copied over by default. By specifying an input like sampleNotCollected which specifies all the values of the patient, this test is invalidated and no longer performing its intended function. Your input should leave the second page unaltered

File: niger/test/tasks/investigation.spec.js [hunk 3]
```diff
@@ -84,10 +85,10 @@ describe('Investigation task', () => {
           expect(investigationValues[`alt_${key}`]).to.deep.eq(expectedDetails[key], key);
         }
 
-        const expectedSymptoms = contactReportForm.report.fields.g_symptoms;
+        /*const expectedSymptoms = contactReportForm.report.fields.g_symptoms;
```
Comment (kennsippell) [inline niger/test/tasks/investigation.spec.js:88]: intentional?

File: niger/app_settings.json (hunk 1 above)
Comment (senseibara) [reply niger/app_settings.json:312]: yes

File: niger/test/tasks/investigation.spec.js (hunk 2 above)
Comment (senseibara) [reply niger/test/tasks/investigation.spec.js:66]: That's correct, i'll update

File: niger/test/tasks/investigation.spec.js (hunk 3 above)
Comment (senseibara) [reply niger/test/tasks/investigation.spec.js:88]: yes !! i'll delete the comment as well

File: niger/test/tasks/investigation.spec.js [hunk 4]
```diff
@@ -83,12 +84,6 @@ describe('Investigation task', () => {
         for (const key of Object.keys(expectedDetails).filter(key => !expectedDetailExceptions.includes(key))) {
           expect(investigationValues[`alt_${key}`]).to.deep.eq(expectedDetails[key], key);
         }
-
-        const expectedSymptoms = contactReportForm.report.fields.g_symptoms;
```
Comment (kennsippell) [inline niger/test/tasks/investigation.spec.js:87]: no symptoms?

File: niger/test/form-inputs.js [hunk 5]
```diff
@@ -44,14 +44,33 @@ module.exports = {
...
   },
   investigation: {
-    unmodifiedInvalid: [[],  ['invalid']],
+    unmodifiedInvalid: [['invalid']],
+    unmodifiedValid: [
+      ['valid'],
+      [],
+      ['poe','lieu','2000-01-01','suspected_case'],
+      ['no','samu','followup_agent'],
+      [Array(1).fill(true),'2000-01-01','no',Array(1).fill(true),'no'],
```
[the remark this answers is not in the document]
Comment (senseibara) [reply niger/test/form-inputs.js:53]: what was i thinking 🤦🏽‍♂️

File: niger/test/tasks/investigation.spec.js (hunk 4 above)
Comment (senseibara) [reply niger/test/tasks/investigation.spec.js:87]: The symptoms from samu to Regional riposte will help on prioritizing the alerts ( Red , ambre etc... ), the investigator will make a proper and more complete diagnosis.

**Comment by senseibara** [comment]
@kennsippell i'm not sure why one of the check is not being completed, still in waiting since yesterday. do you have an idea ?

**Comment by kennsippell** [comment]
@senseibara Looks like a flakey integration with Travis? Unsure of what that was... Looks fixed?

**Pull request merged**